concepts/key-concepts.md

```diff
@@ -62,74 +66,79 @@ streams.
 
 ## Filtering
 
-In some cases it is required to perform modifications on the Events content, the process to alter, enrich or drop Events is called Filtering.
+You might need to perform modifications on the Event's content. The process to alter,
+append to, or drop Events is called [_filtering_](data-pipeline/filter.md).
 
-There are many use cases when Filtering is required like:
+Use filtering to:
 
-* Append specific information to the Event like an IP address or metadata.
-* Select a specific piece of the Event content.
-* Drop Events that matches certain pattern.
+- Append specific information to the Event like an IP address or metadata.
+- Select a specific piece of the Event content.
+- Drop Events that match a certain pattern.
 
 ## Tag
 
-Every Event that gets into Fluent Bit gets assigned a Tag. This tag is an internal string that is used in a later stage by the Router to decide which Filter or Output phase it must go through.
+Every Event ingested by Fluent Bit is assigned a Tag. This tag is an internal string
+used in a later stage by the Router to decide which Filter or
+[Output](data-pipeline/output.md) phase it must go through.
 
-Most of the tags are assigned manually in the configuration. If a tag is not specified, Fluent Bit will assign the name of the Input plugin instance from where that Event was generated from.
+Most tags are assigned manually in the configuration. If a tag isn't specified,
+Fluent Bit assigns the name of the [Input](data-pipeline/input.md) plugin
+instance where that Event was generated from.
 
 {% hint style="info" %}
-The only input plugin that **does NOT** assign tags is [Forward](../pipeline/inputs/forward.md) input. This plugin speaks the Fluentd wire protocol called Forward where every Event already comes with a Tag associated. Fluent Bit will always use the incoming Tag set by the client.
+The [Forward](../pipeline/inputs/forward.md) input plugin doesn't assign tags. This
+plugin speaks the Fluentd wire protocol called Forward where every Event already
+comes with a Tag associated. Fluent Bit will always use the incoming Tag set by the
+client.
 {% endhint %}
 
-A Tagged record must always have a Matching rule. To learn more about Tags and Matches check the [Routing](data-pipeline/router.md) section.
+A tagged record must always have a Matching rule. To learn more about Tags and
```

Comment:
```suggestion
A tagged record must always have a `Matching` rule. To learn more about Tags and
```
not sure if a code block is the best approach here but I feel like something is needed because otherwise the capitalization by itself sticks out